test/cholesky/lkjcholesky.jl

```diff
@@ -135,12 +135,14 @@ using FiniteDifferences
             @test m isa Cholesky{eltype(d)}
             @test Matrix(m) ≈ I
         end
-        @test_broken partype(LKJCholesky(2, 4f0)) <: Float32
+        @test partype(LKJCholesky(2, 4f0)) <: Float32
```

Comment:
We should add tests for the other cases as well, e.g.,
```suggestion
        for (d, η) in ((2, 4), (2, 1), (3, 1)), T in (Float32, Float64)
            @test @inferred(partype(LKJCholesky(d, T(η)))) === T
        end
```